sse_webserver.py

```diff
@@ -0,0 +1,122 @@
+import json
+from twisted.internet import interfaces, reactor, defer, task
+from FeedQueue import FeedQueue
+from twisted.web import server, resource, error
+
+class SSEServer(resource.Resource):
+    connections = {}
+    isLeaf = True
+    looping_calls = {}
+
+    def render_GET(self, request):
+        request.responseHeaders.addRawHeader(b"content-type",
+                                             b"application/json")
+        if self.is_valid_path(request.postpath):
+            self.handle_request(request)
+            request.notifyFinish().addErrback(self.responseFailed,
+                                              request, request.postpath)
+            # reactor.callLater(10, request.finish)
+            return server.NOT_DONE_YET
+        else:
+            return self.invalid_request(request=request,
+                                        code=403,
+                                        reason="Invalid Path")
+
+    def responseFailed(self, err, request,
+                       key=["user", "bob.id.fedoraproject.org"]):
+        print('Connection was disconnected from ' + str(request) +
+              'removing from active connections')
+        self.remove_connection(request, key=key)
+
+    def is_valid_path(self, path=["user", "bob.id.fedoraproject.org"]):
```

Comment:
Having a default list as part of the function definition isn't a good idea.

Cf http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments